.github/workflows/github-ci.yml

```diff
@@ -0,0 +1,46 @@
+name: GitHub Actions Demo
+run-name: ${{ github.actor }} is testing out GitHub Actions 🚀
+on:
+  pull_request:
+    branches:    
+      - master
+  push:
+    branches:
+      - master
+    
+jobs:
+  pr_and_merge_ci:
+    runs-on: ubuntu-latest
```

Comment:
Is it possible to lock this to a specific version? `latest` may introduce unexpected update that may break the build accidentally